docs/group-user-management.rst

```diff
@@ -0,0 +1,227 @@
+administrator privileges. We will show:
+
+- How to manage groups, creating and editing a new/existing group
+- How to manage users, creating and editing a new/existing user
+
+**Resources:**
+--------------
+
+-  Documentation:
+
+   -  https://docs.openmicroscopy.org/latest/omero/sysadmins/server-permissions.html
+
+   -  https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html
+
+   -  https://docs.openmicroscopy.org/omero/latest/sysadmins/cli/usergroup.html
+
+
+Setup:
+------
+
+No setup needed for OMERO.web administration panel (see Web Interface chapter below) except working OMERO.web.
+
+**Command Line interface installation**
+
+The installation instructions can be
+found at \ https://docs.openmicroscopy.org/latest/omero/users/cli/installation.html\ .
+
+
+**Step-by-step:**
+-----------------
+
+Administrate using the Web Interface
+------------------------------------
+
+Below are some useful links to know more about permissions and administrator privileges:
+
+-  https://docs.openmicroscopy.org/latest/omero/sysadmins/server-permissions.html
+
+-  https://docs.openmicroscopy.org/latest/omero/sysadmins/restricted-admins.html
+
+1. In your web browser, go to the server address provided.
+
+2. Login using the username and password provided.
+
+3. In the top toolbar, click the Admin button |image0|\ .
+
+4. Managing Groups:
+
+   a. Click on the Groups tab.
+
+   b. You can search for groups if desired
+
+   c. To create a new Group:
+
+      i.   Click on the Add new Group button
+
+      ii.  The Name and Permissions fields are mandatory.
+
+      iii. Click Save
+
+   d. To edit a Group:
+
+      iv.  Click on the Pencil button |image1|
+
+      v.   You can add or remove members or group’s owners. Change permissions etc..
+
+      vi.  Make sure that the data owned by a user is moved or transferred to another user before removing the user from the group
+
+      vii. Click Save
+
+5. Managing Users:
+
+   e. Click on the Users tab
+
+   f. You can search for users if desired
+
+   g. How to identify users:
+
+      viii. Users with administrators privileges with \ |image2|
+
+      ix.   Active users with\ |image3|
+
+      x.    Inactive users with |image4|
+
+      xi.   LDAP users with |image5|
+
+   h. To create a new User:
+
+      xii.  Click on the Add new User button
+
+      xiii. Mandatory fields are highlighted in red
+
+      xiv.  You can select the role of the user
+
+      xv.   Click Save
+
+   i. To edit a User:
+
+      xvi.   Click on the Pencil button |image6|
+
+      xvii.  You can add or remove the user to/from a group. Modify the roles etc..
+
+      xviii. Click Save
+
+   j. Creating an administrator privileges allow to give some limited rights to some trusted users e.g. to allow a facility manager to import data for other users. It is currently preferable to create users with such role via the Web Interface.
```

Comment:
```suggestion
   j. Creating an administrator with privileges allows to give some limited rights to some trusted users e.g. to allow a facility manager to import data for other users. It is currently preferable to create users with such roles via the Web Interface.
```
I think this is an improvement but not quite sure what you were going for so I'm just guessing.